automation/taskcluster/lib/tasks.py

```diff
@@ -161,6 +163,50 @@ def craft_test_task(self, variant):
             },
         )
 
+   
+    def craft_ui_tests_task(self):
+        artifacts = {
+            "public": {
+                "type": "directory",
+                "path": "/build/fenix/results",
+                "expires": taskcluster.stringDate(taskcluster.fromNow(DEFAULT_EXPIRES_IN))
+            }
+        }
+
+        env_vars = {
+            "GOOGLE_PROJECT": "moz-fenix",
+            "GOOGLE_APPLICATION_CREDENTIALS": ".firebase_token.json"
+        }
+
+        gradle_commands = (
+            './gradlew --no-daemon clean assembleArmDebug assembleArmDebugAndroidTest',
+        )
+
+        test_commands = (
+
+            'chmod +x automation/taskcluster/androidTest/ui-test.sh',
+            'automation/taskcluster/androidTest/ui-test.sh arm -1',
```

Comment:
For now we only use `arm` here`? Just curious since this patch also includes some x86 configurations.

Reply:
yes, we decided to run the UI tests (for now) only on arm since we didn't want the flakiness of x86 devices failing builds.  We leave a config file for x86 in place in the event we should ever like to run them.  Flank can also be run locally for anyone with a GCP and firebase account who might want to run this config (same for tablet)